schema/deploy/shipping/views.sql

```diff
@@ -2091,19 +2093,74 @@ create or replace view shipping.genome_submission_metadata_v1 as
           when identifier_set.name in ('collections-kiosks', 'collections-kiosks-asymptomatic') then 'Shelters'
           when identifier_set.name in ('collections-school-testing-home', 'collections-school-testing-observed') then 'Snohomish Schools'
           when identifier_set.name in ('collections-seattleflu.org') then 'SCH'
-          when identifier_set.name in ('collections-uw-home', 'collections-uw-observed') then 'HCT'
+          when identifier_set.name in ('collections-uw-home', 'collections-uw-observed', 'collections-uw-tiny-swabs') then 'HCT'
           when identifier_set.name in ('collections-radxup-yakima-schools-home', 'collections-radxup-yakima-schools-observed') then 'Yakima Schools'
           else 'SFS'
         end as source,
         location.hierarchy -> 'puma' as puma,
+        -- Convert census tract to counties for Washington counties
+        -- May be removed if county is added to `location.hierarchy`
+        case left(location.hierarchy -> 'tract', 5)
+          when '53001' then	'Adams'
+          when '53003' then	'Asotin'
+          when '53005' then	'Benton'
+          when '53007' then	'Chelan'
+          when '53009' then	'Clallam'
+          when '53011' then	'Clark'
+          when '53013' then	'Columbia'
+          when '53015' then	'Cowlitz'
+          when '53017' then	'Douglas'
+          when '53019' then	'Ferry'
+          when '53021' then	'Franklin'
+          when '53023' then	'Garfield'
+          when '53025' then	'Grant'
+          when '53027' then	'Grays Harbor'
+          when '53029' then	'Island'
+          when '53031' then	'Jefferson'
+          when '53033' then	'King'
+          when '53035' then	'Kitsap'
+          when '53037' then	'Kittitas'
+          when '53039' then	'Klickitat'
+          when '53041' then	'Lewis'
+          when '53043' then	'Lincoln'
+          when '53045' then	'Mason'
+          when '53047' then	'Okanogan'
+          when '53049' then	'Pacific'
+          when '53051' then	'Pend Oreille'
+          when '53053' then	'Pierce'
+          when '53055' then	'San Juan'
+          when '53057' then	'Skagit'
+          when '53059' then	'Skamania'
+          when '53061' then	'Snohomish'
+          when '53063' then	'Spokane'
+          when '53065' then	'Stevens'
+          when '53067' then	'Thurston'
+          when '53069' then	'Wahkiakum'
+          when '53071' then	'Walla Walla'
+          when '53073' then	'Whatcom'
+          when '53075' then	'Whitman'
+          when '53077' then	'Yakima'
+          else null
+        end as county,
         -- If location is null, assume the sample is from within Washington.
```

Comment:
Is it still appropriate to assume the sample is from WA when the above CASE block results in NULL?

Reply:
As far as I know, SFS/SCAN only tests samples within Washington state. 
Even if we don't have detailed location data, I think it's safe to assume it's from WA.

Reply:
Sounds good to me, I'll mark it resolved and approve away.